Library/Formula/fakeroot.rb

```diff
@@ -23,11 +23,35 @@ class Fakeroot < Formula
     sha1 "a30907ffdcfe159c2ac6b3f829bd5b9a67188940"
   end
 
+  # This patch handles mapping the variadic arguments to the system openat to
+  # the fixed arguments for our next_openat function.
+  patch %(diff --git a/libfakeroot.c b/libfakeroot.c
+index 15fdd1d..29d738d 100644
+--- a/libfakeroot.c
++++ b/libfakeroot.c
+@@ -2446,6 +2446,6 @@ int openat(int dir_fd, const char *pathname, int flags, ...)
+         va_end(args);
+         return next_openat(dir_fd, pathname, flags, mode);
+     }
+-    return next_openat(dir_fd, pathname, flags);
++    return next_openat(dir_fd, pathname, flags, NULL);
+ }
+ #endif
+)
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!